### PR TITLE
update fabric repo to be the same as office-ui-fabric-react.js

### PR DESCRIPTION
**Pull request by Caseyyu2**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Include a change request file using `$ npm run change`

#### Description of changes

(give an overview)

#### Focus areas to test

(optional)

### CODE REVIEW

File: packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx [hunk 1]
```diff
@@ -940,6 +944,19 @@ export class ComboBox extends BaseComponent<IComboBoxProps, IComboBoxState> {
     );
   }
 
+  @autobind
+  private _onCalloutRendered() {
```
Comment (sebraddo) [inline packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx:948]: Why are we making the combobox callout focusable?

File: packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx [hunk 2]
```diff
@@ -1025,6 +1042,7 @@ export class ComboBox extends BaseComponent<IComboBoxProps, IComboBoxState> {
         onMouseEnter={ this._onOptionMouseEnter.bind(this, item.index) }
         onMouseMove={ this._onOptionMouseMove.bind(this, item.index) }
         onMouseLeave={ this._onOptionMouseLeave }
+        onKeyDown={ this._onListItemKeyDown(item.index) }
```
Comment (sebraddo) [inline packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx:1045]: I think this bit of code reintroduces the issue where you have to tab through each item in the combobox to get to the next element.  Not sure that we should ever have tab focus on the combobox elements

**Review by sebraddo — COMMENTED** [review]
Looks good, left a few comments about accessibility/tabbing issues

File: packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx (hunk 2 above)
Comment (Caseyyu2) [reply packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx:1045]: My PR is trying to close the gap between fabric repo js and control repo js. This is the bit they added. Jiahong might have better idea

**Pull request merged**